apps/mobile/TheApp/TheCoin/Balances.cs

```diff
@@ -12,37 +12,37 @@ public class Balances
 		private Logger logger = LogManager.GetCurrentClassLogger();
 
 		public ulong MainBalance;
-		public ulong TapCapBalance;
+		//public ulong TapCapBalance;
 
-		public double ExchangeRate => fxRate?.Buy.GetValueOrDefault(1) * fxRate?._FxRate.GetValueOrDefault(1) ?? 1;
-		public ulong TotalBalance => MainBalance + TapCapBalance;
+		public double ExchangeRate => fxRate?.Buy * fxRate?._FxRate ?? 1;
+		public ulong TotalBalance => MainBalance /*+ TapCapBalance*/;
```

Comment:
With the availability of some Eth L2 scaling networks, the TapCap escrow account will (hopefully) no longer be needed